src/components/Permissions/DataPermissions/__snapshots__/DataPermissions.spec.jsx.snap

```diff
@@ -168,6 +165,9 @@ exports[`DataPermissions should display DataPermissions 1`] = `
               </div>
             </div>
           </li>
+          <hr
+            class="MuiDivider-root MuiDivider-inset"
```

Comment:
Is it intended that the divider also moved with the alphabetical ordering? Shoudln't it still be between `Alan` connector and `Contacts` app?

This seems to be related to `isNotLastItem()` that use an unsorted array for compararison